packages/clerk-js/src/ui/signIn/SignInStart.tsx

```diff
@@ -36,28 +46,60 @@ export function _SignInStart(): JSX.Element {
   const instantPassword = useFieldState('password', '');
   const [error, setError] = React.useState<string | undefined>();
 
-  const standardFormAttributes = userSettings.enabledFirstFactorIdentifiers;
-  const web3FirstFactors = userSettings.web3FirstFactors;
-  const socialProviderStrategies = userSettings.socialProviderStrategies;
-  const passwordBasedInstance = userSettings.instanceIsPasswordBased;
```

Comment:
This PR seems to remove the instant password fix. Should we wait for UserSettings or should we implement a fix for the AuthConfig version? cc @SokratisVidros

Reply:
We should keep the fix despite the revert